src/tools/rbd/action/DiskUsage.cc

```diff
@@ -180,9 +180,7 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
           goto out;
         }
 
-        if (snapname != NULL) {
-          total_prov += snap->size;
-        }
+        total_prov += snap->size;
```

Comment:
I agree most, but this 'hidden' is really strange in <total> item, especially the 'used' is even bigger.
The word 'provisioned' means it maybe not a real size and not free , so I think this 'hidden' is not a necessary one, instead, it confused others.